commercial/app/model/commercial/events/LiveEventMembershipInfo.scala

```diff
@@ -0,0 +1,22 @@
+package model.commercial.events
+
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+
+case class LiveEventMembershipInfo(id: String,
+                                   url: String,
+                                   mainImageUrl: String)
+
+object LiveEventMembershipInfo {
+  implicit val format = Json.format[LiveEventMembershipInfo]
+
+  // based on play.api.libs.json.LowPriorityDefaultReads.traversableReads
+  implicit val formats = new Reads[Seq[LiveEventMembershipInfo]] {
```

Comment:
This is entirely my fault, but this should be called `readsLiveEventMembershipInfo`, not `formats` as it isn't a `Formats` 😕